src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -281,7 +281,7 @@ private void executeCommand(String commandText) {
      * Called to update AutoComplete when new commands.
      */
     public void updateCommandAutoComplete(String commandText) {
-        if (!commandText.isBlank() && !commandText.endsWith(" ")) {
```

Comment:
This caused a bug where no command was eagerly evaluated if the command ends with space, such as 'patient 0  '

Reply:
This one is not a bug. It does not make sense to eager evaluate on for additional space.

In a sense, we should specify, eager evaluation will not be run if there is no meaningful changes to the command in command box.